docs/components/modeler/web-modeler/create-a-process-application.md

```diff
@@ -0,0 +1,63 @@
+---
+id: create-a-process-application
+title: Create a process application
+description: Create a process application.
+---
+
+import EmptyProjectImg from './img/process-applications/empty-project.png'
+import ReassignMainProcessImg from './img/process-applications/reassign-main-process.png'
+import FileListImg from './img/process-applications/file-list.png'
+
+Get started by creating a new process application.
```

Comment:
Would add a purpose here -- for example, "Create a new process application to *insert purpose*."

Reply:
Thanks - added 👍